docs/tex/tutorials/probabilistic-pca.tex

```diff
@@ -0,0 +1,87 @@
+\newcommand{\R}{ \mbox{${\Bbb R}$}}
+
+
+\title{Probabilistic PCA}
+
+\subsection{Probabilistic PCA}
+
+Probabilistic PCA is useful when dealing with data that is dependent on both principal axes and latent variables \citep{Tipping99probabilisticprincipal}. It is often used when there are missing values in the data.
+
+We'll run through a quick tutorial to implement it in Edward below. The full script can be accessed
+\href{https://github.com/blei-lab/edward/blob/master/examples/probabilistic_pca.py}
+{here}.
+
+\subsubsection{Data}
+
+We simulate our data points below. We'll talk about the individual variables and what they stand for in the next section. For this example, $\mathbf{x}\in\mathbb{R}^2$.
+\begin{lstlisting}[language=Python]
+def build_toy_dataset(N, D, K, sigma=1):
+  x_train = np.zeros((D, N))
+  w = np.random.normal(0.0, 2.0, size=(D, K))
+  z = np.random.normal(0.0, 1.0, size=(K, N))
+  mean = np.dot(w, z)
+  for d in range(D):
+    for n in range(N):
+      x_train[d, n] = np.random.normal(mean[d, n], sigma)
+
+  return x_train
+
+N = 5000  # number of data points
+D = 2  # data dimensionality
+K = 1  # latent dimensionality
+
+x_train = build_toy_dataset(N, D, K)
+\end{lstlisting}
+
+This generates a dataset that looks like this:
+
+\includegraphics[width=450px]{/images/ppca-fig0.png}
+
+\subsubsection{Model}
+
+Consider a dataset $X = \{(\mathbf{x}_1, \mathbf{x}_2,\ldots , \mathbf{x}_n)\}$ where $\mathbf{x}_i \in \Bbb{R}^D$.
```

Comment:
Following the code, I prefer the notation that `N` denotes the data set size and `n` denotes an index to one data point.